### PR TITLE
CM-87: Define GPB proto file

### PR DESCRIPTION
**Pull request by DKlimenko**
Define GPB API between SDL and IVDCM

### CODE REVIEW

**Comment by DKlimenko** [comment]
@bonewell

**Comment by DKlimenko** [comment]
@VladSemenyuk

**Comment by DKlimenko** [comment]
@ESaenko

**Comment by VladSemenyuk** [comment]
Reviewed

File: src/components/ivdcm_module/interface/rpc.proto [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     ON_INTERNET_STATE = 0;
+     SEND_IVDCM_DATA = 1;
+  }
+  required MessageType rpc_type = 1;
+  required SDLRPCName  rpc_name = 2;
+  required bytes       params   = 3;
+}
+
+message OnInternateStateNotificationParams {
+  required string nic_name = 1;
```
Comment (bonewell) [inline src/components/ivdcm_module/interface/rpc.proto]: What means this parameter?

**Comment by bonewell** [comment]
Reviewed.

File: src/components/ivdcm_module/interface/rpc.proto (hunk 1 above)
Comment (DKlimenko) [inline src/components/ivdcm_module/interface/rpc.proto]: Network interface controller name.  Netwok name like sdlproxy1

**Pull request merged**